server/src/test/java/io/crate/integrationtests/CopyFromFailFastITest.java

```diff
@@ -136,7 +136,7 @@ public Sessions sqlOperations() {
         Path target = Files.createDirectories(tmpDir.resolve("target"));
         int numDocs = 100;
 
-        var indexMetadata = clusterService().state().getMetadata().index("t");
+        var indexMetadata = clusterService().state().getMetadata().index(sqlExecutor.getCurrentSchema() + ".t");
```

Comment:
minor, just for nicer code: can we have a `getCurrentSchema()` method directly in IntegTestCase instead?